gui/wxpython/lmgr/frame.py

```diff
@@ -263,6 +264,14 @@ def show_menu_errors(messages):
             self.GetMapDisplay().Raise()
         wx.CallAfter(self.Raise)
 
+        # if demolocation, add demo layer to map layer tree
+        if is_current_mapset_in_demolocation():
+            ltype, lname, = get_demolocation_layer_info()
```

Comment:
This function could be avoided.

`lname="country_boundaries@PERMANENT"`

Reply:
Changed.